test/iifCompareTests.jl

```diff
@@ -56,7 +56,7 @@ using Test
     # fg2 has been solved, so it should fail on the estimate dictionary
     @test !compareSimilarVariables(fg, fg2, skipsamples=true, skip=Symbol[:initialized;:inferdim])
 
-    tree = wipeBuildNewTree!(fg2)
+    tree = solveGraph!(fg2)
```

Comment:
you could also just build tree with resetBuildTree!(fg)

Reply:
yeah I couldn't find out what the new name was - thanks!

Reply:
also the return from 'tree, smt, hist = solveTree!(fg)' is a Tuple

Reply:
maybe we should consider a better name 'buildTreeReset!(fg)' which is more probable and better fits verbNouns